electives/80405.md

```diff
@@ -7,7 +7,9 @@
 | Exams |   None   |
 
 Game theory is a philosophy department course that served as an introductory to basic game theories. The main topics include Minimax Theory, Nash Equilibrium, Sequential Games, Bayesian Games, and Bargaining Theory. Scenarios include zero-sum games, auctions, votes, prisoner's dilemma, and sequential games. Although the class informally assumes some knowledge about decision theory, it is an introductory class, and all of the decision theory parts are easy to understand, as it is not in-depth. There are no prerequisites for this class, nor is there a requirement for a solid mathematic background.
+
 ## Lectures
+
```

Comment:
If you're going to add the spaces to this header, please do it to the rest of the doc as well